packages/client-adapters/src/errors/adapter-error.js

```diff
@@ -13,14 +13,6 @@ class AdapterError extends Error {
       // Continue if environment does not support captureStackTrace.
     }
   }
-
-  update (config) {
-    this.adapter = `${config.category}.${config.adapterName}`
-    this.methodName = config.method
-
-    this.message = `${this.message} (${this.adapter}.${this.methodName})`
-    return this
-  }

```

Comment:
I did not find it to be used anywhere so I've decided to remove the unused code. Please let me know if this code is needed and I will return it back.

Reply:
That's a good idea, I will do that.